src/types/community/comment.ts

```diff
@@ -20,7 +21,8 @@ export interface CommentProps {
   user_profile_image: string;
   created_at: string;
   updated_at?: string;
-  is_like: boolean;
   like_count: number;
-  children?: ChildrenCommentProps[];
+  is_like: boolean;
+  deleted: number;
+  childComments?: ChildCommentProps[];
```

Comment:
```suggestion
  oneComments?: oneCommentProps[];
```
>소 단위 기능의 컴포넌트는 card, item, list, one 과 같이 뜻을 정확히 구분합니다.

와 같이 컴포넌트의 props로 전달이 되는 부분은 좀 더 보기 편한 props 명을 사용하면 좋을 것 같아요. 만약 childComments가 컴포넌트가 아닌 props 변수라면 `child_comments`로 스네이크 표기법으로 사용해도 될 것 같습니다.

Reply:
API 명세 key-value를 그대로 차용했는데 생각해보니 `childComment`만 카멜케이스로 작성이 되어 있군요!

이 부분 담당 벡엔드분과 논의 후 수정하도록 하겠습니다 👍